### PR TITLE
Complete c-style-checker task

### PR DESCRIPTION
**Pull request by Epremyan-Norik**
381706-4

### CODE REVIEW

**Comment by TolyaTalamanov** [comment]
LGTM 👍

File: main.cpp [hunk 1]
```diff
@@ -18,13 +18,29 @@ using namespace clang::tooling;
...
-        // Your code goes here
-    };
+  CastCallBack(Rewriter& rewriter) : rewriter_(rewriter) {};
 
     virtual void run(const MatchFinder::MatchResult &Result) {
         // Your code goes here
+        const auto *CastExpr = Result.Nodes.getNodeAs<CStyleCastExpr>("cast");
+        SourceManager &SourceMng =  *Result.SourceManager;
+
+        if (CastExpr->getExprLoc().isMacroID())
```
[the remark this answers is not in the document]
Comment (Epremyan-Norik) [reply main.cpp:28]: Maybe I should check only one other condition: CastExpr==nullptr?

[the remark this answers is not in the document]
Comment (TolyaTalamanov) [reply main.cpp:28]: I hope you can manage it for yourself

[the remark this answers is not in the document]
Comment (Epremyan-Norik) [reply main.cpp]: commit changes

**Pull request closed without merging**